addon_service/osf_models/fields.py

```diff
@@ -0,0 +1,132 @@
+import datetime as dt
+from decimal import Decimal
+
+import jwe
+from dateutil.parser import isoparse
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.db.models import JSONField
+
+from app.settings import (
+    OSF_SENSITIVE_DATA_SALT,
+    OSF_SENSITIVE_DATA_SECRET,
+)
```

Comment:
```suggestion
from django.conf import settings
```
django settings modules aren't meant to be imported directly

(alternately, could import `from app.env` and skip adding them to `app.settings`)